_posts/2025-04-21-OpenSearch-Benchmark---The-Journey-Ahead.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  OpenSearch Benchmark - The Journey Ahead
```

Comment:
This is more a "how to get started" than a "journey ahead". If you want it to be road ahead, you need to condense the first 2/3 of the post that covers what it is and how to use it, then expand the bullet points to include details of what OSB is building, and why.

If we need an introductory post, that's fine, but it can be under separate cover. I think it would be great to have that! Then we can have this post about what's coming.

Reply:
We plan several other posts on various upcoming features, so renamed this to "An Update".